backends/platform/ios7/ios7_video.mm

```diff
@@ -422,6 +423,9 @@ - (id)initWithFrame:(struct CGRect)frame {
 
 	[self setupGestureRecognizers];
 
+	if (@available(iOS 14.0, *)) {
```

Comment:
I forgot to indicate here that after my compilation tests from today, this was not an issue for me. So I am guessing this would only be an issue for very old Xcode versions.

Reply:
We have a history of supporting old platforms. However that happens when we have a dedicated developer spending time to keep ScummVM working on those. And even then they might require a modern compiler (we at least require support for c++11).

For iOS we don't even have a maintainer for modern iOS and Xcode versions, let alone old ones. So I don't have an issue with dropping support for using very old Xcode version that do not support the `@available` keyword (I suspect that would mean any version older than Xcode 9, which is probably the first version to support it since it was released in 2017).